requirements.txt

```diff
@@ -17,15 +17,18 @@ django-phone-field==1.8.0
 django-storages==1.9.1
 docopt==0.6.2
 docutils==0.15.2
+googlemaps==4.2.0
```

Comment:
pia googlemaps at the moment I don't think if we have working google maps

Reply:
Okey on it

Reply:
awesome